kopf/_cogs/configs/conventions.py

```diff
@@ -181,7 +221,7 @@ def make_v2_key(self, key: str, max_length: int = 63) -> str:
         return final_key
 
     def make_suffix(self, key: str) -> str:
-        digest = hashlib.blake2b(key.encode('utf-8'), digest_size=4).digest()
```

Comment:
Frankly speaking, I do not even remember why it is blake2b here, since a usual md5 or sha256 would be more than fine. But it is as it is.